lib/libutee/include/pta_system.h

```diff
@@ -13,6 +13,17 @@
 #define PTA_SYSTEM_UUID { 0x3a2f8978, 0x5dc0, 0x11e8, { \
 			 0x9c, 0x2d, 0xfa, 0x7a, 0xe0, 0x1b, 0xbe, 0xbc } }
 
+/*
+ * Having keys with too few bits impose a potential security risk, hence set a
+ * lower bound (i.e., 128 bits).
+ */
+#define TA_DERIVED_KEY_MIN_SIZE 16
+
+/* Same value as max in huk_subkey_derive */
+#define TA_DERIVED_KEY_MAX_SIZE 32
+
+#define TA_DERIVED_EXTRA_DATA_MAX_SIZE 1024
```

Comment:
very minor: somewhat long but maybe prefix with `TA_DERIVED_KEY_` instead of `TA_DERIVED_`.

Reply:
Hmm, the `TA_DERIVED_` was supposed to be the "prefix" and `KEY_...` and `EXTRA_...` was supposed to the things that differ. Note that this is the common theme in the code etc, i.e., I have nowhere written `foobar_key` ... `foobar_key_extra`.

Reply:
ok. actually  `TA_DERIVED_KEY_EXTRA_DATA_MAX_SIZE` would a pretty large macro name, not very handy...

Reply:
Ok, ignoring this idea.